t/modules/Compare/Custom.t

```diff
@@ -11,6 +11,10 @@ ok(!$fail->verify(got => "anything"), "always fails");
 
 is($pass->operator, 'CODE(...)', "default operator");
 is($pass->name, '<Custom Code>', "default name");
+ok(!$pass->stringify_got, "default stringify_got");
+
+my $stringify = $CLASS->new(code => sub { 1 } , stringify_got => 1 );
+ok($stringify->stringify_got, "custom stringify_got()");
 
```

Comment:
It would be nice if you also added a test that verifies the stringification in a custom check.

Reply:
I added a failing test which should stringify but I am not enough in the internals of Test2-Suite to know how to test the test. Can you give me hint how to do it or which other test file is a good example?